handler/CMakeLists.txt

```diff
@@ -44,33 +44,41 @@ if (ANDROID)
     )
 endif (ANDROID)
 
-add_executable(handler main.cc)
-target_sources(handler PRIVATE ${CRASHPAD_HANDLER_LIBRARY_FILES})
-target_compile_features(handler PUBLIC cxx_std_17)
-target_include_directories(handler PRIVATE ..)
-target_link_libraries(handler PRIVATE client tools compat snapshot minidump)
+if (HANDLER STREQUAL "INTEGRATED")
```

Comment:
Instead of having the user set a separate, likely hard to find CMake variable, maybe create the static library unconditionally, or maybe conditioned on `ANDROID` instead, along with a comment explaining how is intended to be used. Something like,

```
# Used to implement a work-around for launching the Crashpad handler stored in an APK.
if (ANDROID)
    add_library(handlerlib STATIC ${CRASHPAD_HANDLER_LIBRARY_FILES})
    target_compile_features(handlerlib PUBLIC cxx_std_17)
    target_link_libraries(handlerlib PRIVATE client tools compat snapshot minidump)
    target_include_directories(handlerlib PUBLIC ..)
endif(ANDROID)
```

This would require updating the name used in the backtrace-android commit:
```
target_link_libraries(backtrace-native handlerlib)
```